stock_by_warehouse/tests/test_stock_available_unreserved.py

```diff
@@ -124,12 +124,12 @@ def test01_stock_levels(self):
         self.compare_qty_available_not_res(product_a, 0)
         self.compare_qty_available_not_res(self.template_ab, 0)
 
-        self.assertFalse(product_a.warehouses_stock)
+        self.assertEqual(product_a.warehouses_stock, "{}")
```

Comment:
Even with the Json field the assertFalse didn't worked @luisg123v

Reply:
I only want to ask: Why doesn't this happen in `stock_by_warehouse_sale`?

Reply:
because it doesn't have test